module/zfs/dbuf.c

```diff
@@ -148,10 +148,17 @@ dbuf_stats_t dbuf_stats = {
 		continue;						\
 }
 
+static uint64_t dirty_writes_lost;
+
+#define	DEBUG_COUNTER_INC(x)
+#define	DEBUG_REFCOUNT_DEC(x)
```

Comment:
These look like dead code at this point.  They served a purpose at one time though.  Maybe just replace with DTrace SDTs?